.changes/unreleased/tinymce-TINY-12578-2025-08-19.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Deprecated
+body: Deprecated 'content_css_cors' config option.
```

Comment:
didn't I explain in slack that we should point to the `crossorigin` setting as the replacement 🤔